src/include/duckdb/main/capi/header_generation/apis/v1/unstable/new_expression_functions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "unstable_new_expression_functions",
+  "description": "API to create and manipulate expressions",
+  "entries": [
+    "duckdb_destroy_expression",
+    "duckdb_expression_return_type",
+    "duckdb_is_constant_expression",
+    "duckdb_constant_expression_value",
+    "duckdb_bind_argument_count",
+    "duckdb_bind_argument_at"
```

Comment:
`duckdb_scalar_function_bind_get_argument_count` and `duckdb_scalar_function_bind_get_argument`